Debian/bin/lib/Cyrus/Docker/Command/build.pm

```diff
@@ -18,6 +18,7 @@ sub opt_spec {
   return (
     [ 'recompile|r', 'recompile, make check, and install a previous build' ],
     [ 'no-cunit|n',  'do not run make check' ],
+    [ 'with-sphinx|s', 'enable sphinx docs' ],
```

Comment:
Conflicted.  I think this should also use `!` form, like `sphinx|s!`.  Or, at least, I think you should consider not calling it `with-sphinx` because that calls to mind the autoconf `--with-X` semantics, which it doesn't really follow.  It's okay as is, but I imagine wanting to tweak it later, myself…

Reply:
Blarg, I'm going to not think about it for now and leave it as is. Feel free to change if you like